prems-resources/clinical-data/src/main/resources/SLING-INF/content/apps/cards/mailTemplates/bodyTemplate.header.html

```diff
@@ -0,0 +1,80 @@
+<!doctype html>
+<html>
+  <head>
+    <title>UHN Canadian Patient Experience Survey</title>
+    <meta http-equiv="content-type" content="text/html;charset=utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <style>
+      .body {
+        max-width: 600px;
+        margin: 32px auto;
+        padding: 0 16px;
+        font-family: Arial, Helvetica, sans-serif;
+        color: #333;
+        line-height: 1.5em;
+      }
+      a {
+        color: #0b6bb5;
+      }
+      p {
+        margin: 1em 0;
+      }
+      hr {
+        margin: 2em 0;
+        opacity: .3;
+      }
+      img {
+        max-width: 100%;
+      }
+      .header {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
```

Comment:
Sounds like this styling is being discarded by some email apps. We could make the `header` a full width `<table>` instead (see next comment).
```suggestion
        width: 100%;
```

Reply:
Nevermind the table, looks like a simple `float: right` on the 2nd image will do the trick.